Client/src/Pages/Details/index.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import CustomizedTables from '../../Components/CustomizedTables';
+import { Box, Typography, useTheme } from '@mui/material';
+
+/**
+ * Details page component displaying monitor details and related information.
+ * @component
+ */
+const DetailsPage = () => {
+  const theme = useTheme();
+
+  // Example monitor data (replace with actual data or fetch from API)
+  const monitorData = {
+    "_id": "66863aa2437936b46e225d1f",
+    "userId": "66863a8f437936b46e225d1a",
+    "name": "Google",
+    "description": "Google",
+    "status": false,
+    "type": "http",
+    "url": "https://www.google.com/404",
+    "isActive": true,
+    "interval": 60000,
+    "createdAt": "2024-07-04T06:01:06.527Z",
+    "updatedAt": "2024-07-04T06:02:00.250Z",
+    "__v": 0,
+    "checks": [
+      {
+        "_id": "66863ad8437936b46e225d6d",
+        "monitorId": "66863aa2437936b46e225d1f",
+        "status": false,
+        "responseTime": 144,
+        "expiry": "2024-07-04T06:02:00.247Z",
+        "statusCode": 404,
+        "createdAt": "2024-07-04T06:02:00.248Z",
+        "updatedAt": "2024-07-04T06:02:00.248Z",
+        "__v": 0
+      },
+      {
+        "_id": "66863b14437936b46e225d79",
+        "monitorId": "66863aa2437936b46e225d1f",
+        "status": false,
+        "responseTime": 140,
+        "expiry": "2024-07-04T06:03:00.167Z",
+        "statusCode": 404,
+        "createdAt": "2024-07-04T06:03:00.168Z",
+        "updatedAt": "2024-07-04T06:03:00.168Z",
+        "__v": 0
+      }
+    ]
+  };
+
+  /**
+   * Function to calculate uptime duration based on the most recent check.
+   * @param {Array} checks Array of check objects.
+   * @returns {string} Uptime duration in hours, minutes, and seconds.
+   */
+  const calculateUptimeDuration = (checks) => {
+    if (!checks || checks.length === 0) {
+      return 'N/A'; 
+    }
+
+    // Sort checks by createdAt descending to get the most recent check first
+    checks.sort((a, b) => new Date(b.createdAt) - new Date(a.createdAt));
```

Comment:
Checks should already be in sorted order when returned from backend I beleive

Reply:
Sounds good will get rid of this bit.